taca_ngi_pipeline/cli.py

```diff
@@ -68,15 +68,24 @@ def deliver(ctx, deliverypath, stagingpath, uppnexid, operator, stage_only, forc
 			  envvar='STATUS_DB_CONFIG',
 			  type=click.File('r'),
 			  help='Path to statusdb-configuration')
+@click.option('--order-portal',
+			  default=None,
```

Comment:
So I check the behavior: if the env variable is not present the option is set to None, with or without the default. If the env variable is present then the option takes the env variable value.
So everything is as expected, I like to force values into variables with default instead of hoping in some nice behavour from the library so I would prefer to leave the `default=None`